doc/build-instructions/cmake.md

```diff
@@ -36,3 +48,5 @@ just to update the `rpath` (on applicable platforms) and to move the binary
 to the corresponding source directory. We do this purely for compatability with
 existing build infrastructure that expects the binaries to be laid out as they
 would be under a `makefile` driven build.
+
+:pencil: For more CMake info see [OMR CMake documentation](https://github.com/eclipse/omr/blob/master/doc/BuildingWithCMake.md)
```

Comment:
Can you move this up above the `## FAQ`?  Right now it gets lots in the FAQ question